test/lomc.jl

```diff
@@ -296,10 +297,20 @@ using Statistics
             @test df2.norm ≈ df3.norm
             @test df3 == df4
 
+            # ReportToFile with skipping interval  
+            df5 = df1[10:10:100,:]
+            r_strat = ReportToFile(filename="test-report.arrow", reporting_interval=10, io=devnull, chunk_size=10)
+            df = lomc!(H, copy(dv); r_strat, laststep=100).df
+            @test isempty(df)
+            df6 = RimuIO.load_df("test-report-3.arrow")
+
+            @test df6 == df5
```

Comment:
```suggestion
            @test df6.shift ≈ df5.shift
            @test df6.norm ≈ df5.norm
```
It looks like the tests are failing on this line. I'm not sure why as the numbers in the DataFrame look the same. You could try to inquire further, but otherwise the suggestion above might fix the test.

Reply:
It worked sometimes, local and on github. Let's see if this test works